components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/PostAuthnMissingClaimHandler.java

```diff
@@ -255,6 +256,12 @@ protected PostAuthnHandlerFlowStatus handlePostAuthenticationForMissingClaimsReq
                 if (log.isDebugEnabled()) {
                     log.debug("Redirecting to outside to pick mandatory claims");
                 }
+                if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                    diagnosticLogBuilder.resultMessage("Redirecting to outside to pick mandatory claims from the " +
+                            "user.")
+                            .resultStatus(DiagnosticLog.ResultStatus.SUCCESS);
```

Comment:
shouldn't we have the application name in all the logs for this flow?

Reply:
+ the tenant domain

Reply:
The application name is already added to this when the diagnosticBuilder object gets created (L197). The same object is getting reused in all flows for simplicity.

Reply:
`the tenant domain`
Do we need to add the tenant domain, because the application belongs to the current tenant, right? From user's point of view, he already knows the tenant's details. So do we need to add that data in the log as well?